src/clients/SpokePoolClient.ts

```diff
@@ -51,7 +52,7 @@ const FILL_DEPOSIT_COMPARISON_KEYS = [
 const FIXED_LOOKBACK_EVENTS = ["FundsDeposited", "FilledRelay", "RelayedRootBundle", "ExecutedRelayerRefundRoot"];
 
 export class SpokePoolClient {
-  private currentTime: number;
+  private currentTime = 0;
```

Comment:
Does it make sense to define this in the constructor? Also, could we use something like:
```suggestion
  private currentTime = Math.floor(Date.now() / 1000);
```

Reply:
Yeah I think we might check for `0` in other places, but I do think this should always be referencing the `SpokePool` block timestamp. Another idea is since we check of `SpokePoolClient.isUpdated` in other places, we might want to set `isUpdated = SpokePool.getCurrentTime()` in the update method

Reply:
I also think that a timestamp could be a good idea, because `.isUpdated` (as it's currently used) isn't well-suited to looping mode. I think we can incorporate an update to this as part of the UBA work.